llvm/utils/git/github-automation.py

```diff
@@ -131,10 +139,23 @@ def run(self) -> bool:
             patch_link = f"\nPatch is {human_readable_size(len(patch))}, truncated to {human_readable_size(DIFF_LIMIT)} below, full version: {self.pr.diff_url}\n"
             diff_stats = diff_stats[0:DIFF_LIMIT] + "...\n<truncated>\n"
```

Comment:
```suggestion
            import html
            diff_stats = html.escape(diff_stats[0:DIFF_LIMIT]) + "...\n<truncated>\n"
```

Should avoid the rendering issues as seen here https://github.com/llvm/llvm-project/pull/66037#issuecomment-1714859370

Reply:
Thanks, I'll test this out.

Reply:
@cor3ntin See https://github.com/llvm/llvm-project/pull/66037#issuecomment-1716167092 It looks slightly better, but it still tries to render some things as markdown.

Reply:
I think i did not look that carefully enough.

we need to escape `patch`, not diff_stats

```diff
- diff_stats += "\n<pre>\n" + patch
+ diff_stats += "\n<pre>\n" + html.escape(patch)
```

But there are other rendering bugs I think, we should limit the number of change files, and the number of lines in the diff as 2 different things. I suppose we can fix that later

Reply:
It looks much better now with escaping the patch: https://github.com/llvm/llvm-project/pull/66037#issuecomment-1716234197

Reply:
Seems like the wrong suggestion was integrated (the first one and not what @cor3ntin fixed in the latest comment above)

Reply:
Fixing in https://github.com/llvm/llvm-project/pull/66322